doc/iterations/2023.6/shapes/shape Migration flow.adoc

```diff
@@ -0,0 +1,29 @@
+= (appetite) Shape
+
+== Problem
+
+In order to include a test which covers the all odesign to view migration path, we will migrate the existing flow model.
+
+This view model should be generated using the soon to be introduced Java APIs to create view models. https://github.com/eclipse-sirius/sirius-components/issues/1919
+
+== Key Result
+
+The flow odesign model should be converted to a view model. All concepts used in the flow model should have a respective view concept.
+
+== Solution
+
+Use the soon to be introduced Java APIs to create view models. https://github.com/eclipse-sirius/sirius-components/issues/1919
```

Comment:
- Where will this file be located? in which project? In which Git repository? https://github.com/ObeoNetwork/FlowDesigner like the existing flow.odesign file? How can I reuse it in other applications using flow already?
- Will it depend on existing flow projects to retrieve the meta-model? (I think so)
- Will it depend on existing flow projects to retrieve existing Java services? (I don't think so)